packages/component/src/Composer.js

```diff
@@ -310,7 +310,8 @@ const Composer = ({
     dispatch
   ]);
 
-  const patchedSelectVoice = useCallback(selectVoice || defaultSelectVoice.bind(null, { language: locale }), [
+  const patchedSelectVoice = useMemo(() => selectVoice || defaultSelectVoice.bind(null, { language: locale }), [
```

Comment:
Why `locale` is removed? I think it's used in the function.

Reply:
not sure what you mean about `locale`. this was a linting fix, I believe. `useCallback` -> `useMemo`

Reply:
Oh right. I reversed the side. I thought `locale` was *removed*. My bad.